parm/post/mom6_interp.csv

```diff
@@ -0,0 +1,18 @@
+SST remapbil MOM6_INTERP_NC 0
+SSS remapbil MOM6_INTERP_NC 0
+SSH remapbil MOM6_INTERP_NC 0
+speed remapbil MOM6_INTERP_NC 0
+MLD_003 remapbil MOM6_INTERP_NC 0
+so remapbil MOM6_INTERP_NC 0
+temp remapbil MOM6_INTERP_NC 0
+latent remapbil MOM6_INTERP_NC 0
+sensible remapbil MOM6_INTERP_NC 0
+SW remapbil MOM6_INTERP_NC 0
+LW remapbil MOM6_INTERP_NC 0
+evap remapbil MOM6_INTERP_NC 0
+lprec remapbil MOM6_INTERP_NC 0
+LwLatSens remapbil MOM6_INTERP_NC 0
+Heat_PmE remapbil MOM6_INTERP_NC 0
+SSU,SSV remapbil MOM6_INTERP_NC 1 cos_rot,sin_rot
+uo,vo remapbil MOM6_INTERP_NC 1 cos_rot,sin_rot
+taux,tauy remapbil MOM6_INTERP_NC 1 cos_rot,sin_rot
```

Comment:
To be consistent with the NCL/Python remapping programs, The following variables should use conservative remapping (referred to as remapcon in CDO):

1. latent
2. sensible
3. SW
4. LW
5. evap 
6. lprec
7. fprec
8. LwLatSens
9. Heat_PmE
10. taux,tauy